### DISCUSSION TITLE
Question about ''premiumUuid'' feature

### DISCUSSION THREAD

**Discussion by zSteley** [games647/FastLogin#743] [Q&A] [answered]
Hi :D

I have a question about premiumUuid feature! 

My network is semi-premium (I allow to join premium and cracked players) 

For now I have premiumUuid set to false on my network, but sometimes it create issues with luckperms... so i plan to change premiumUuid to true. 

If I do premiumUuid: true, Will I have issues with my spigot plugins that uses uuids of players to get information, save progress, etc? 

Thanks!!

**Comment by TuxCoding** [comment, marked as answer]
>If I do premiumUuid: true, Will I have issues with my spigot plugins that uses uuids of players to get information, save progress, etc

Yes, because it will change and so all the associated data like inventory gets disconnected.

**Reply by zSteley** [reply]
Okey, so what are the steps to use premiumuuids on my network??

**Reply by TuxCoding** [reply]
Just enable it, but as UUIDs should be unique to a player's identity, they might get a new inventory profile.

**Comment by zSteley** [comment]
Okey thanks!!

Also I have this issue sometimes in bungeecord console, is normal? 
https://paste.md-5.net/ligugoruwe.md

**Reply by TuxCoding** [reply]
No, do you use a proxy. The error means an abrupt end of the transmission.

**Reply by zSteley** [reply]
Yes i'm using Travertine, so, what I need to change to avoid this error?

**Reply by TuxCoding** [reply]
It's either a Mojang server or connectivity issue. If you use proxies, check their stability. It's like sudden cut of the connection.
